homework/src/index.js

```diff
@@ -31,17 +33,116 @@
   }
 
   function main(url) {
-    fetchJSON(url, (err, data) => {
-      const root = document.getElementById('root');
-      if (err) {
-        createAndAppend('div', root, { html: err.message, class: 'alert-error' });
-      } else {
-        createAndAppend('pre', root, { html: JSON.stringify(data, null, 2) });
-      }
+    const root = document.getElementById("root");
+
+    const div = createAndAppend("div", root);
+    const header = createAndAppend("p", div, {
+      text: "HYF Repositories",
+      class: "header"
     });
+
+    const table = createAndAppend("table", root, { class: "table" });
+    const select = createAndAppend("select", header, { class: "select" });
+
+    const tbody = createAndAppend("tbody", table);
+    const tr = createAndAppend("tr", tbody);
+    const container = createAndAppend("div", root, { class: "container" });
+    createAndAppend("td", tr, { text: "Repository: ", class: "label" });
+    const repositoryLink = createAndAppend("a", tbody);
+    const description = createAndAppend("tr", tbody);
+    const forks = createAndAppend("tr", tbody, { class: "label" });
+
+    const updated = createAndAppend("tr", tbody, { class: "label" });
+
```

Comment:
By looking to the [documentations](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tbody) under the example table you will find a blue table that contains the Permitted content header aside, there you can find the valid values you can use.

Until this poit you have HTML generated:
```javascript
<table class="table">
  <tbody>
    <tr>
      <td class="label">Repository: </td>
    </tr>
    <a></a>
    <tr></tr>
    <tr class="label"></tr>
    <tr class="label"></tr>
  </tbody>
</table>
```
and by validating with [w3c validator](https://validator.w3.org/nu/#textarea) your code we will notice that you have here:
![screenshot_2018-09-04 showing results for contents of text-input area - nu html checker](https://user-images.githubusercontent.com/25272822/45041613-08c54200-b069-11e8-99ff-5c55baed20ed.png)

So in order to fix this the only thing you have to do is just to move your `a` tag into a `td` that is inside of a `tr` tag so you will have the following:
```javascript
<table class="table">
  <tbody>
    <tr>
      <td class="label">Repository: </td>
    </tr>
    <tr>
      <td>
        <a></a> <!-- Like so -->      
      </td>
    </tr>
    <tr></tr>
    <tr class="label"></tr>
    <tr class="label"></tr>
  </tbody>
</table>
```